### PR TITLE
Fix display issues with unsubscribe modal & update kano-nav site-root based on whether the user is logged in

### PR DESCRIPTION
**Pull request by abridger**
Details in individual commits.

### CODE REVIEW

File: src/elements/kw-app/kw-app.html [hunk 1]
```diff
@@ -254,7 +261,9 @@
                 this.cacheOrAPI(this._getUrl('stats')).then(stats => {
                     this.stats = stats;
                 });
-                this._checkNewsletterSubscription();
+                this.async(() => {
+                    this._checkNewsletterSubscription();
+                }, 0);
```
Comment (vikdoro) [inline src/elements/kw-app/kw-app.html]: would this work?
 this.async(() => this._checkNewsletterSubscription());

Comment (abridger) [reply src/elements/kw-app/kw-app.html]: Yes! Good point.

File: gulpfile.js [hunk 2]
```diff
@@ -57,7 +57,7 @@ gulp.task('dev', (done) => {
 gulp.task('watch', function() {
     browserSync.init({
         server: {
-            baseDir: "./www",
+            baseDir: "./src",
```
Comment (paulvarache) [inline gulpfile.js]: Since a recent merge, the watch task need to serve from the `www` folder. Is this a PR conflict?

Comment (abridger) [reply gulpfile.js]: OK, sorry I can remove that – it wasn't working for me locally, which is why I changed it. I guess I just hadn't pulled in the latest changes. Will revert now.

**Comment by abridger** [comment]
@vikdoro I also noticed that the unsubscribe modal was clashing with the onboarding modal, so added a check for the unsubscribe modal in `_needOnboardingModal`. There's probably a better way of preventing modal conflicts long-term though.

**Pull request merged**